.github/workflows/visual-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: Visual Regression Testing
+
+on: [pull_request]
+
+jobs:
+    build:
+
+        runs-on: ubuntu-latest
+        services:
+            mysql:
+                image: mysql:5.7
+                ports:
+                    - 3306
+
+        steps:
+            - uses: actions/checkout@v1
+              with:
+               token: ${{ secrets.STAABM_TOKEN }}
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v1
+              with:
+                  php-version: 7.3
+                  extensions: intl, imagick
+                  coverage: none # disable xdebug, pcov
+                  tools: cs2pr
+            - name: Get Composer Cache Directory
+              id: composer-cache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+            - name: Cache dependencies
+              uses: actions/cache@v1
+              with:
+                  path: ${{ steps.composer-cache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                  restore-keys: ${{ runner.os }}-composer-
+            - name: Install Dependencies
+              run: composer install --prefer-dist
+            - run: |
+                  mysql -uroot -h127.0.0.1 -proot -e 'create database redaxo5;'
+                  git apply .github/workflows/default.config.yml.github-action.diff
+            - run: |
+                  php redaxo/src/addons/tests/bin/setup.php
+                  php redaxo/bin/console user:create myusername mypassword --admin
```

Comment:
@bloep ich kann mich mit diesem user nicht im backend anmelden (geht auch nicht lokal bei mir)... fehlt hier noch was?

Reply:
problemursache ist, dass `logintries` beim neuen user `NULL` ist, statt `0`

Reply:
fix in https://github.com/redaxo/redaxo/pull/3465